src/Host.h

```diff
@@ -934,25 +934,25 @@ private slots:
     // 1 is Fixed red color ring with adjustable outer/inner diameter
     // 2 is fixed blue/yellow colors ring with adjustable outer/inner diameter
     // 3 is adjustable outer(primary)/inner(secondary) colors ring with adjustable outer/inner diameter
-    quint8 mPlayerRoomStyle;
-    QColor mPlayerRoomOuterColor;
-    QColor mPlayerRoomInnerColor;
+    quint8 mPlayerRoomStyle = 0;
+    QColor mPlayerRoomOuterColor{Qt::red};
+    QColor mPlayerRoomInnerColor{Qt::white};
     // Percentage of the room size (actually width) for the outer diameter of
     // the circular marking, integer percentage clamped in the preferences
     // between 200 and 50 - default 120:
```

Comment:
Glad to see comments reduced, because code is obvious by itself

```suggestion
    // between 200 and 50:
```

Reply:
Actually that value reflected the size of the original mark that was enhanced to allow the colours to be changed.

Reply:
Come again?

Reply:
The original marker (before I "enhanced" it) happened to have a size that corresponded to the 120 value...

Reply:
I can't follow, I am afraid.